nbconvert/preprocessors/execute.py

```diff
@@ -85,6 +85,20 @@ class ExecutePreprocessor(Preprocessor):
         )
     )
 
+    raise_on_iopub_timeout = Bool(
+        False, config=True,
+        help=dedent(
+            """
+            If `False` (default), then the kernel will continue waiting for
+            iopub messages until it receives all of them. If `True`, then an
```

Comment:
This doesn't seem quite right - if False, it will stop waiting when it hits the fixed timeout.

(Also, there's not really an 'all of them' for iopub messages - the client can't know that there are no more to come, it just makes an intelligent guess that a kernel status idle message means it can stop waiting. If the notebook does things with threads, there could be more messages after that.)